ast-names.h

```diff
@@ -33,7 +33,7 @@ AST_CHILD(TYPE, 0)
 
 /* 1 child node */
 AST_CHILD(VAR, 1)
-AST_CHILD(CONST, 1)
+/* AST_CHILD(CONST, 1) */
```

Comment:
That sorta breaks any code which uses constant declarations.

Would it work if you put something like this at the top of the header?

```
#ifdef CONST
# undef CONST
#endif
```